CyberSource/utilities/flex/utils/transient_token_generator.py

```diff
@@ -0,0 +1,19 @@
+from . import base64
```

Comment:
Put this file directly under `CyberSource/utilities/flex`

Reply:
Changes are done